vignettes/wip/dm-db.Rmd

```diff
@@ -146,10 +187,18 @@ my_dm_total$total_loans %>%
 
 ## Persisting results
 
-- Relevant phrase, from `?dbplyr::compute`: "Force computation of a query", needs to be adapted to our needs (=multiple queries)
-- Requires write access to the database (at least for temporary tables), which we don't have on relational.fit
-    - Workaround for demonstration only: copy to local SQLite database, see below for details on `copy_dm_to()`
+After adding columns to link tables or calculating summaries of data, we might want these changes to the database to persist. 
+#FIXME - if compute stores results in temporary tables, how are they made permanent or inserted into the database itself?
+
+To force a dm to execute the SQL query generated by the operations it has performed, we use the `compute()` function, originally implemented in {dbplyr}, but reimplimented in {dm} to provide the same functionality.
+#FIXME - what's the right language here? inherited? carried over from?
 
+`compute()` forces the computation of a query, in this case the query or multiple queries created by the dm to represent all operations that have been performed but not yet evaluated.
+The results are stored in temporary tables on the database server.
+Calling `compute()` requires write access, otherwise an error is returned.
+
+relational.fit.cvut.cz does not allow write access. 
+As a workaround to demonstrate the usage of `compute()`, we will copy our dm into a local SQLite database and move forward with that as the source of our dm.
```

Comment:
I forgot to mention, we now can

```r
my_dm_sqlite <- dm_financial_sqlite()
```

Do we still need it? Maybe discuss `copy_dm_to()` at the end with an example?